book/installation.rst

```diff
@@ -6,210 +6,206 @@
+To efficiently use the assets shipped with the OroCRM, it is recommended to install node.js. Detailed installation 
+instructions are available in the `node.js installation document`_.
 
-node.js
-~~~~~~~
+Step 1. Get the Source Code and Define Dependencies
+---------------------------------------------------
 
-To efficiently use the assets shipped with the OroCRM, it is recommended to install node.js. Please
-refer to the `node.js installation document`_ for detailed instructions.
+First, you need to obtain the application's source code and define the dependencies. There are two ways to do so: 
 
-Getting the Source Code
------------------------
+**1a.** :ref:`Clone the GitHub repository <book-installation-github-clone>` and run the *composer install* command.
 
-First, you need to prepare the installation process by obtaining the application's source code and
-installing all necessary dependencies. You can get the source code in two different ways (both
-examples assume that you want to install the application into the ``/var/www/orocrm`` directory):
+**1b.** :ref:`Download the source code archive <book-installation-download-archive>` and run the *composer update* command.
 
-#. :ref:`Clone the GitHub repository <book-installation-github-clone>`.
-#. :ref:`Download the source code archive <book-installation-download-archive>`.
 
 .. _book-installation-github-clone:
 
-1. Clone the GitHub Repository
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Clone the GitHub Repository
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You can clone the official `GitHub repository`_ to obtain the source code
-and checkout the release you want to use:
+**1a.1.** Go to the directory, to which you want to save the OroCRM folder ([$folder_location]) 
 
+**1a.2.** Use the *git clone* command. Specify the version to download. (In the example, it is "1.8.2").
+ 
 .. code-block:: bash
 
-    $ cd /var/www
+    $ cd [$folder_location]
     $ git clone -b 1.9.0 https://github.com/orocrm/crm-application.git orocrm
 
-.. note::
+.. hint:
 
     Along with ``1.9.0``, you can use any other released version or even the master branch to run
     the latest development version of the OroCRM.
 
-.. sidebar:: Installing the OroPlatform Application
+.. sidebar:: Get Source Code for the OroPlatform Application
 
-    Use the `Platform application repository URL`_ instead if you do not want to install the full
-    CRM:
+    If you are installing the Platform application, use the `Platform application repository URL`_ :
 
     .. code-block:: bash
 
-        $ cd /var/www
+        $ cd [$folder_location]
         $ git clone -b 1.9.0 https://github.com/orocrm/platform-application.git orocrm
 
-Next, you'll need to install the necessary dependencies which may take some time:
+**1a.3.** Go to the "orocrm" directory and run the *composer install* command, in order to install the dependencies. 
 
 .. code-block:: bash
 
+    $ cd orocrm
     $ composer install
```

Comment:
Recommend to use composer install --prefer-dist --no-dev . In other case user get error or all dev environment will be installed